compiler/codegen/CodeGenRA.cpp

```diff
@@ -700,18 +698,10 @@ TR::Register * OMR::CodeGenerator::allocateRegister(TR_RegisterKinds rk)
    self()->addAllocatedRegister(temp);
    if (self()->getDebug())
       self()->getDebug()->newRegister(temp);
-   if (  rk == TR_GPR
-      && (  !self()->getDebug()
-         || !self()->getTraceRAOption(TR_TraceRASpillTemps)
-         || performTransformation(self()->comp(), "O^O SPILL TEMPS: Set UpperHalfIsDead on %s\n", self()->getDebug()->getName(temp)))) // allocateRegister is called for the vmthread during initialization when getDebug has not been initialized yet
-      {
-      temp->setIsUpperHalfDead();
```

Comment:
These weren't used anywhere so we removed them. It had a few dependencies on some x86 code which was dead as a result.